src/pages/guide/index.md

```diff
@@ -1,8 +1,256 @@
 ---
-title: Application Testing Guide | Commerce Testing
-description:
+title: Application Testing Guide Introduction | Commerce Testing
+description: Learn about different testing strategies to ensure code quality for Adobe Commerce and Magento Open Source projects.
 ---
 
-# Application Testing Guide
+# Application Testing Guide introduction
 
-Lorem ipsum dolor sit amet
+The Commerce framework actively leverages various testing strategies to ensure product and code quality. Product quality matters to the end-users working with the storefront, admin panel, web APIs and CLI. Product quality tests verify that the user gets expected results while interacting with the system.
+
+Code quality is important to developers working with the codebase including:
+
+-  System integrators
+-  Extension developers
+-  Adobe core team
+-  Community contributors
+
+Code quality matters for:
+
+-  Extensibility - It should be straightforward to extend or modify existing behavior. Good extensibility allows for:
+   -  Customizations and extensions using modularity of the platform.
+   -  Evolution of the platform with new releases.
+-  Maintainability - It should be straightforward for developers to work within the system. It can be improved with:
+   -  Complexity management - Reduce the learning curve and risk of adding new bugs.
+   -  Automatic bug detection - Reduces total cost of ownership.
+   -  Coding standards - Ensures consistency and readability.
+
+Automated tests are required by [definition of done][] for any code changes.
+
+## Product Quality Tests
+
+-  [Functional Testing Framework][] - Tests the storefront and admin panel UI.
+-  [Web API Functional][] - Tests the REST, SOAP, and GraphQL areas.
+-  [Integration][] - Ensures that all the parts work together seamlessly.
+-  Performance - Tracks changes in CPU, Memory and other metrics. Executed nightly for develop branches. Custom builds can be configured using [performance toolkit][].
```

Comment:
Just an FYI: this guide has a lot of level 2 headings that are immediately followed by ordered or unordered lists, which causes the spacing to be a bit weird. Probably can't fix this now, but might be something we should address in the future.

Reply:
Yeah, I think that'll have to be a style fix in the theme.